particles/Tutorial/Kotlin/Demo/src/pt3/TTTGame.arcs

```diff
@@ -4,7 +4,7 @@
 schema Person
   name: Text
   avatar: Text
-  id: Number
+  idx: Number
```

Comment:
I've made this name change in the 3 relevant TTT demos, since the underlying Entity now has an ID field by default. I don't feel 100% I've gone in the right direction with this, but at least things build now...

Reply:
Eep!

Can you file a bug to fix this? You should be allowed to write handles named "id". We need to come up with a solution for this..